composables/transaction/transactionMintDrop.ts

```diff
@@ -0,0 +1,23 @@
+import { MintDropParams } from './types'
+
+export function execMintDrop({
+  item,
+  api,
+  executeTransaction,
+  isLoading,
+}: MintDropParams) {
+  const { accountId } = useAuth()
+
+  const args = item.nfts.map((allocatedNft, index) =>
+    api.tx.nfts.mint(item.collectionId, allocatedNft.id, accountId.value, {
+      ownedItem: (item.availableSerialNumbers || [])[index] || null,
```

Comment:
`(item.availableSerialNumbers || [])[index]`

could this error if `index` is out of bounds?

Reply:
No error. If out of bounds, it returns undefined and falls back to `null` finally.